### DISCUSSION TITLE
Wheres my CoPilot trial gone?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#121276] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

So i signed up for copilot and input payment details for 30 day trial. I go into VS code to activate and taken to page that doesnt load, then when i try to troubleshoot on GitHub i am prompted to pay $10 today for access to co pilot. What happened to my trial?

**Comment by davecheney** [comment, marked as answer]
@sineurdev it looks like your card was declined during the pre payment phase. 
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
